### PR TITLE
Improve custom hover 

### PR DESCRIPTION
**Pull request by sandy081**
Implements the feedback provided in the issue - https://github.com/microsoft/vscode/issues/118077

- Ability to show hover pointer
- Ability to show hover on left or right positions
- Ability to show compact hover

@Tyriar Sorry for last minute request. Is it possible to get it done today? Would like to have this available for testing.

### CODE REVIEW

File: src/vs/workbench/services/hover/browser/media/hover.css [hunk 1]
```diff
@@ -14,6 +14,63 @@
...
+	height: 5px;
+}
+
+.monaco-workbench .workbench-hover-pointer.left {
+	left: -3px;
+}
+
+.monaco-workbench .workbench-hover-pointer.left:after {
+	-moz-transform: rotate(135deg);
+	-webkit-transform: rotate(135deg);
```
Comment (Tyriar) [inline src/vs/workbench/services/hover/browser/media/hover.css:44]: `transform` on the bottom?

**Review by Tyriar — APPROVED** [review]
I didn't review too thoroughly but it seems to work and not regress so good to go 👍

File: src/vs/workbench/services/hover/browser/media/hover.css (hunk 1 above)
Comment (sandy081) [reply src/vs/workbench/services/hover/browser/media/hover.css:44]: Done

**Comment by sandy081** [comment]
Thanks. Merged.

**Pull request merged**